im-commons/src/main/java/com/wipro/ats/bdre/ResolvePath.java

```diff
@@ -26,11 +23,12 @@
  * Created by AS294216 on 1/5/2016.
  */
 public class ResolvePath {
-    private static Logger LOGGER = Logger.getLogger(ResolvePath.class);
-
 
     private static Pattern pattern = Pattern.compile("%(.)");
 
+    private ResolvePath(){
+    }
+
     public static String replaceVars(String line) {
         StringBuffer sb = new StringBuffer();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace the synchronized class "StringBuffer" by an unsynchronized one such as "StringBuilder". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1149)